testing/kuttl/e2e/password-change/04-errors.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: password-change-pguser-password-change
+data:
+  uri: ""
```

Comment:
⛏️ Similar here. We set `uri` to blank then have blank in the errors file. I believe I understand it now, but it is initially counterintuitive to me.

If I understand correctly, PGO will eventually place the specified `password` into the `uri` field.